platforms/FreeBSD/makePackage

```diff
@@ -78,6 +78,23 @@ cd "$CURRENT_DIRECTORY"
 SWIFT_VERSION="$("$SWIFT_TOOLCHAIN_HOME/bin/swift" --version | grep -Eo 'version [0-9.]+' | sed 's/version //')"
 echo "Swift version being packaged is $SWIFT_VERSION"
 
+# Get the currently installed version for all of the packages that we need.
+# This script assumes that it was run on the build machine and therefore
+# that the packages on this machine correspond to those Swift was built
+# against.
+get_package_version()
+{
+  echo "$(pkg info "$1" | grep Version | sed 's/Version.*: //')"
+}
+
+LIBUUID_PKG_VERSION=$(get_package_version libuuid)
+LIBFFI_PKG_VERSION=$(get_package_version libffi)
+LIBXML2_PKG_VERSION=$(get_package_version libxml2)
+MPDECIMAL_PKG_VERSION=$(get_package_version mpdecimal)
+PYTHON_PKG_VERSION=$(get_package_version python311)
+READLINE_PKG_VERSION=$(get_package_version readline)
+SQLITE3_PKG_VERSION=$(get_package_version sqlite3)
```

Comment:
Do we still need SQLite? I thought we used swift-toolchain-sqlite now and thus do not need the system vended SQLite.

Reply:
Well the idea for swift-toolchain-sqlite is that it's for systems without a native package management system to provide sqlite... meaning Windows and Android.

So far on Darwin we've been using the system SQLite and on Linux we've still been using the copy provided by the package manager... I expect it would make sense for FreeBSD to do the same as Linux here.